test/snowpack/runtime/runtime.test.js

```diff
@@ -159,4 +159,24 @@ describe('runtime', () => {
       await fixture.cleanup();
     }
   });
+
+  it('Can handle folders with dots', async () => {
+    const fixture = await testRuntimeFixture({
+      'public/example.com/index.html': `<html></html>\n`,
+      'public/example/index.html': `<html></html>\n`,
+      'snowpack.config.js': `
+        module.exports = {
+          mount: {
+            public: { url: '/', static: true }
+          }
+        }
+      `,
+    });
+    try {
+      let contents = (await fixture.loadUrl('/example.com/')).contents.toString('utf8');
```

Comment:
This tests the dev server URL. Running `snowpack build` didn’t seem to trip it up (probably because it was dealing with the full `/example.com/index.html` path)